listenbrainz/webserver/static/js/src/LastFMImporter.tsx

```diff
@@ -194,14 +194,14 @@ export default class LastFmImporter extends React.Component<
       }
     } catch {
       // Retry if there is a network error
-      retry("Error");
+      retry("Network error");
```

Comment:
Yes! That's a much much much better error message, nice catch!
Unfortunately, it looks like the error message isn't used at all. Maybe worth having a look at using it, in another PR?

Reply:
It should appear in the console as part of a warning now.